crates/bevy_gltf/src/lib.rs

```diff
@@ -64,50 +74,81 @@ impl Plugin for GltfPlugin {
 /// Representation of a loaded glTF file.
 #[derive(Asset, Debug, TypePath)]
 pub struct Gltf {
+    /// Unnamed scenes loaded from the glTF file.
```

Comment:
It's not the unnamed scenes, it's all the scenes, named or not. Same for the other fields